code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -207,14 +207,18 @@
 		R.hat = null
 	R.update_headlamp()
 	R.alpha = 0
-	animate(R, alpha = 255, time = 50)
+	animate(R, alpha = 255, time = 30)
+	do_transform_delay()
+
+/obj/item/robot_module/proc/do_transform_delay()
+	var/mob/living/silicon/robot/R = loc
 	var/prev_lockcharge = R.lockcharge
 	R.SetLockdown(1)
 	R.anchored = TRUE
 	sleep(2)
-	for(var/i in 1 to 4)
+	for(var/i in 1 to 3)
 		playsound(R, pick('sound/items/drill_use.ogg', 'sound/items/jaws_cut.ogg', 'sound/items/jaws_pry.ogg', 'sound/items/welder.ogg', 'sound/items/ratchet.ogg'), 80, 1, -1)
-		sleep(12)
+		sleep(5)

```

Comment:
this changes the total transformation time from 5 seconds to 1.7 seconds, and makes the sounds happen *way* too fast.

Reply:
It's the only thing that ended up ensuring there isn't the flicker of a standard borg at the end of the animation cycle.